api/envoy/api/v2/endpoint.proto

```diff
@@ -114,4 +115,28 @@ message ClusterLoadAssignment {
 
   // Load balancing policy settings.
   Policy policy = 4;
+
+  // References endpoint groups by name.
+  repeated Egds endpoint_groups = 6;
```

Comment:
A ClusterLoadAssignment does not recommend sending Endpoints and EndpointGroups together, and while the code implementation can be done, it can cause confusion when applied.

If EGDS is used, I think it is necessary to initialize the cluster after receiving the response from all the endpoint groups. Otherwise, how should the envoy handle the initialization state of the cluster ?